kb/deadline-exceeded-troubleshooting.md

```diff
@@ -22,7 +22,7 @@ The following sections discuss the nature of this error and how to troubleshoot
 
 Cloud users cannot access some of the logs needed to diagnose the source of the error.
 
-If you're using Temporal Cloud, check your Workflow History for error messages and create a [support ticket](https://support.temporal.io/) with Temporal.
+If you're using Temporal Cloud, check your Workflow History for error messages and create a [support ticket](https://docs.temporal.io/cloud/how-to-create-a-ticket-for-temporal-support).
```

Comment:
I'd remove the "check your workflow history for error message". maybe better to say here "create a ticket and provide as much information as possible, including namespace name and workflow id(s) of some executions experiencing the issue" or something similar